lib/main_publiccloud.pm

```diff
@@ -171,6 +200,7 @@ sub load_create_publiccloud_tools_image {
 sub load_publiccloud_cli_tools {
     loadtest 'boot/boot_to_desktop';
     loadtest 'publiccloud/azure_cli';
+    loadtest 'publiccloud/azure_more_cli';
     loadtest 'publiccloud/aws_cli';
     loadtest 'publiccloud/google_cli';
```

Comment:
```suggestion
    if(get_var('AZURE_CLI_TEST_NAME')) {
        loadtest 'publiccloud/azure_more_cli';
    } else {
        loadtest 'publiccloud/azure_cli';
        loadtest 'publiccloud/aws_cli';
        loadtest 'publiccloud/google_cli';
    }
```

Reply:
OK, the variables are now renamed. But:
1) Please documment the variables in `variables.md`
2) Please do the advised changes in `lib/main_publiccloud.md`:
```suggestion
    if(get_var('PUBLIC_CLOUD_AZURE_CLI_TEST')) {
        loadtest 'publiccloud/azure_more_cli';
    } else {
        loadtest 'publiccloud/azure_cli';
        loadtest 'publiccloud/aws_cli';
        loadtest 'publiccloud/google_cli';
    }
```

Reply:
> 
This one is already fixed.

   if (get_var('PUBLIC_CLOUD_AZURE_CLI_TEST')) {
        loadtest 'publiccloud/azure_more_cli';
    } else {
        loadtest 'publiccloud/azure_cli';
        loadtest 'publiccloud/aws_cli';
        loadtest 'publiccloud/google_cli';
    }

Reply:
> OK, the variables are now renamed. But:
> 
> 1. Please documment the variables in `variables.md`
> 2. Please do the advised changes in `lib/main_publiccloud.md`:

I can add the variable in variables.md

Reply:
@pdostal - My suggestion for the remaining tests should not be in else.

`if (get_var('PUBLIC_CLOUD_AZURE_CLI_TEST')) {
loadtest 'publiccloud/azure_more_cli';
} else {
loadtest 'publiccloud/azure_cli';
loadtest 'publiccloud/aws_cli';
loadtest 'publiccloud/google_cli';
}`

We should be able to test other cli's azure, aws, google along with azure_more_cli.

``if (get_var('PUBLIC_CLOUD_AZURE_CLI_TEST')) {
loadtest 'publiccloud/azure_more_cli';
}
loadtest 'publiccloud/azure_cli';
loadtest 'publiccloud/aws_cli';
loadtest 'publiccloud/google_cli';`

Let me know your thoughts, I can change it accordingly.